.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+<!-- (an executive summary of the changes, ideally in one sentence) -->
+This PR...
+
+## Changes
+<!-- (major technical changes list) -->
+
+-
+
+## Checklist
+
+- [ ] tested with Python3.6
+- [ ] run `make check` or `make fix` for the formatting check
+- [ ] signed commits
```

Comment:
In the medium term, I’d like the CI to perform these checks and report the results instead of having them in the issue template.

Reply:
I completely agree - this should not be done by hand. This solely points out that CI is not ready yet for pynitrokey.